deepmedic/neuralnet/trainer.py

```diff
@@ -254,7 +254,7 @@ def _get_mom_from_schedule(self):
             x2 = final_it_for_sch - first_it_for_sch
             x = tf.maximum( tf.constant(0, dtype="float32"), curr_it - first_it_for_sch )
             x = tf.minimum( x, x2 )
-            y_min = self._init_lr_tfv
+            y_min = self._init_mom_tfv
```

Comment:
Changed  hotfixed in master, unrelated to current PR. Should have been pulled from master. There will be conflict.
Easy to solve but just saying. Better do changes only associated to PR and pull the rest.